var/spack/repos/builtin/packages/nwchem/package.py

```diff
@@ -155,3 +158,11 @@ def install(self, spec, prefix):
             with open(".nwchemrc", 'w') as f:
                 f.write(nwchemrc)
             install(".nwchemrc", share_path)
+
+    def setup_run_environment(self, env):
+        env.set('NWCHEM_BASIS_LIBRARY', join_path(
+            self.prefix,
+            'share/nwchem/libraries/'))
+        env.set('NWCHEM_NWPW_LIBRARY', join_path(
+            self.prefix,
+            'share/nwchem/libraryps/'))
```

Comment:
You can use `self.prefix.share.nwchem.libraryps` here, then `join_path` isn't needed. Same above.

Reply:
Sure, I can we listed as package maintainer. 6.8.1 is a maintenance release for 6.8, in the sense that it does not contain any new functionality, but bug fixes, instead

Reply:
The change to `NWCHEM_BASIS_LIBRARY` and `NWCHEM_NWPW_LIBRARY` fails since a forward slash character "/" is needed at the end ... is there any way to get it in place without using `join_path()`?

Reply:
I would like to get a comment about the following line 
```
    depends_on('gcc@7:9', when='@:6', type=('build', 'link', 'run'))
```
The reason behind this line is that gcc>9 is not going to work with NWChem versions prior to 7.0.0
Is there a better way to put this constraint?

Reply:
For the trailing slash, the only other thing you could do is `self.prefix.foo.join('')`, that might work, you'll have to check.

For the GCC conflict, the correct way to do this would be:
```python
conflicts('%gcc@10:', when='@:6', msg='NWChem versions prior to 7.0.0 do not build with GCC 10')
```
Compilers aren't technically dependencies at the moment, although that will change in the future.

Reply:
I could not get anything other than the original ugly `join_path()` to work, therefore I have left it as it is.
I have inserted your `gcc@10:` suggestion, instead.